source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -1,10 +1,12 @@
-# Set up a custom domain using the cdn-route service
+# Managing custom domains using the cdn-route service
 
-This section explains how to configure a custom domain name for your app by using the GOV.UK PaaS `cdn-route` service to set up a content distribution network (CDN) that will route and [cache](/deploying_services/use_a_custom_domain/#caching) requests to your app.
+This section explains how to manage custom domains for your app by using the GOV.UK PaaS cdn-route service to set up a content distribution network (CDN) that routes requests to and [caches](/deploying_services/use_a_custom_domain/#caching) requests from your app.
```

Comment:
"caches requests from your app" doesn't seem right here. I think it should either be "requests to your app", or "responses from your app". The latter is probably more correct as the responses are what is cached.

Reply:
my mistake, @jpluscplusm did say that and I forgot to put it in